src/zoid/buttons/component.jsx

```diff
@@ -262,6 +292,8 @@ export const getButtonsComponent : () => ButtonsComponent = memoize(() => {
                 required: false,
                 default:  () => noop,
                 decorate: ({ props, value = noop }) => {
+                    logLatencyInstrumentationPhase({ buttonSessionID: props.buttonSessionID, phase: 'buttons-first-render' });
```

Comment:
Let's use the `once()` function from belter here since the decorate function can end up getting executed multiple times.